tests/interface.rs

```diff
@@ -1,16 +1,8 @@
-winrt::import!(
-    dependencies
-        os
-    types
-        windows::foundation::{IStringable, Uri}
-);
-
 use winrt::ComInterface;
+use winrt::{IStringable, Uri};
```

Comment:
I wonder if re-exporting the foundation namespace in the generated code would also make sense.

Reply:
I thought having different names/aliases for all those types would be confusing and lead to a mix of code that used one name or the other.

Reply:
I personally think mixing and matching would be unlikely and not that disruptive if it occurs. It's nice to be able to refer to the types in their "native" namespace.

Reply:
Done